src/handlers/domanda.py

```diff
@@ -1,27 +1,34 @@
 from telegram import Update, InlineKeyboardButton, InlineKeyboardMarkup
 from telegram.ext import CallbackContext
+from src.data.costanti import CATEGORIA, LIVELLO, DOMANDA_CORRENTE
 import json
 
 def domanda(update: Update, context: CallbackContext) -> None:
     data = load_file(context)
 
-    domanda_corrente = context.user_data['domanda_corrente']
+    domanda_corrente = context.user_data[DOMANDA_CORRENTE]
     question = data[domanda_corrente]
     testo = question['testo']
     risposte = question['risposte']
+    
+    keyboard = []
+    for i, j in enumerate(risposte):
+        button = InlineKeyboardButton(j['testo_risposta'], callback_data=f'{domanda_corrente}:{i}')
+        keyboard.append([button])
 
-    keyboard = [[InlineKeyboardButton(j['testo_risposta'], callback_data=f'{domanda_corrente}:{i}')] for i, j in enumerate(risposte)]
     reply_markup = InlineKeyboardMarkup(keyboard)
     context.bot.send_message(chat_id=update.effective_chat.id, text=testo, reply_markup=reply_markup)
 
+
 def load_file(context: CallbackContext) -> dict:
-    if context.user_data['livello'] == "1: Facile":
-        f = open('src/data/facile.json', 'r', encoding="utf-8")
-    elif context.user_data['livello'] == "2: Intermedio":
-        f = open('src/data/intermedio.json', 'r', encoding="utf-8")
-    elif context.user_data['livello'] == "3: Difficile":
-        f = open('src/data/difficile.json', 'r', encoding="utf-8")
+    if context.user_data[LIVELLO] == "1: Facile":
+        path = 'facile.json'
+    elif context.user_data[LIVELLO] == "2: Intermedio":
+        path = 'intermedio.json'
+    elif context.user_data[LIVELLO] == "3: Difficile":
+        path = 'difficile.json'
 
-    data = json.load(f)[f"{context.user_data['categoria']}"]
+    f = open(f'src/data/{path}', 'r', encoding="utf-8")
+    data = json.load(f)[f"{context.user_data[CATEGORIA]}"]
```

Comment:
potresti usare `with` in modo tale da effettuare l' `f.close()` automaticamente appena finisci di utilizzare il file
```suggestion
    with open(f'src/data/{path}', 'r', encoding="utf-8") as f:
        data = json.load(f)[f"{context.user_data[CATEGORIA]}"]
```